applications/tari_validator_node/src/lib.rs

```diff
@@ -114,15 +114,15 @@ pub async fn run_validator_node(config: &ApplicationConfig, shutdown_signal: Shu
         .get_or_create_global_db()
         .map_err(|e| ExitError::new(ExitCode::DatabaseError, e))?;
 
-    info!(
-        target: LOG_TARGET,
-        "🚀 Node starting with pub key: {}, address: {}",
-        node_identity.public_key(),
-        node_identity
-            .public_addresses()
-            .first()
-            .ok_or_else(|| ExitError::new(ExitCode::UnknownError, "public address not found for validator node"))?
-    );
+    // info!(
```

Comment:
When initialising a brand new validator node, this will panic as 

```rs
node_identity
            .public_addresses()
            .first()
            .ok_or_else(|| ExitError::new(ExitCode::UnknownError, "public address not found for validator node"))?
```

as `node_identity.public_addresses()` is empty atm